### PR TITLE
User API

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: src/ts/http/StatusCode.ts [hunk 1]
```diff
@@ -1,5 +1,66 @@
 export default class StatusCode {
+  public static get ACCEPTED(): number {
+    return 202;
+  };
+
```
Comment (herrmannplatz) [inline src/ts/http/StatusCode.ts]: why not just use public static constance

```
export default class StatusCode {
  public static ACCEPTED : number = 202;
  public static BAD_GATEWAY : number = 502;
}
```

File: src/ts/auth/AuthAPI.ts [hunk 2]
```diff
@@ -18,39 +18,35 @@ export default class AuthAPI {
...
+    return this.client.sendRequest(config);
   }
 
-  removeCookies(login: LoginData, labels?: string[]): AxiosPromise {
-    const url = this.client.createUrl(`${AuthAPI.URL.COOKIES}/remove`);
+  public postLogin(login: LoginData): Promise<AccessTokenData> {
+    const config: AxiosRequestConfig = {
+      data: {
+        email: login.email,
+        password: login.password.toString()
```
Comment (lipis) [inline src/ts/auth/AuthAPI.ts]: what happened to the trailing commas? :(

File: src/ts/core/WireAPIClient.ts [hunk 3]
```diff
@@ -15,15 +17,25 @@ export default class WireAPIClient {
     client: undefined
   };
 
+  public user: { api: UserAPI } = {
```
Comment (lipis) [inline src/ts/core/WireAPIClient.ts]: `{api: UserAPI}`

File: src/ts/user/UserData.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  public deleted?: boolean;
+  public email?: string;
+  public handle?: string;
+  public id: string;
+  public locale: string;
+  public name: string;
+  public phone?: string;
+
+  constructor(data: any) {
+    Object.assign(this, data);
```
Comment (herrmannplatz) [inline src/ts/user/UserData.ts]: :/

Comment (bennyn) [inline src/ts/user/UserData.ts]: I am open for ideas.

**Comment by bennyn** [comment]
@herrmannplatz, please give me a second chance. 😋

**Pull request merged**